### DISCUSSION TITLE
Import intellisense

### DISCUSSION THREAD

**Discussion by Fractal-Tess** [vercel/turborepo#1693] [Help] [answered]
Hello there,

I'm looking for instructions on how to set up a turborepo with an `assets` package where my static assets would reside.
Now the problem with my current setup is that my vscode IntelliSense is unable to resolve the relative workspace directories and provide me with useful suggestions when doing imports.

For example, take a look at this. I've set up the `assets` workspace as a static directory, and I'm able to import and use files from it, but there is no intellisense:
![image](https://user-images.githubusercontent.com/75957529/184619451-4170e3f9-d97e-4bbd-9363-1994ff980f1d.png)

However, if I do provide the full relative path, I do get intellisense.
![image](https://user-images.githubusercontent.com/75957529/184619546-391016d1-06eb-4ba1-b3b0-76e518664684.png)

Any clue as to how I can get IntelliSense when doing the workspace reference?

**Comment by anthonyshew** [comment, marked as answer]
If I had to guess, the auto-import discovery across packages isn't supported because of limitations in TypeScript. I'm thinking the imports within the package are powered directly by file discovery of VSCode.

Even if my guesses are indeed wrong, Turborepo won't be able to help here as these are things outside of its scope.
